services/users/src/routes/analytics.ts

```diff
@@ -0,0 +1,63 @@
+import express from 'express'
+import useDB from '../providers/db'
+import { query } from 'athena-query'
+
+const router = express.Router()
+
+const { formatResult, getUserById } = useDB()
+
+router.get('/:userId', async (req: express.Request, res: express.Response) => {
+    try {
+        const { userId } = req.params
+        const user = await getUserById(userId)
+        const { accounts } = user
+        const addresses = accounts.map((account) => account.address)
+        const opt = {
+            profile: 'consensus-networks-dev',
```

Comment:
@hawyar messaged you about the profile option, see here

Reply:
@shanejearley - see where? Is the env variable `AWS_PROFILE`?

Reply:
Looks like it. Committing and resolving for now.

Reply:
Sorry here are the details: this works for the local env with AWS config files present. Otherwise (in deployed env), profile is not used and there are no AWS config files present. We have the analytics data publicly accessible right now for ease so it should be fine to remove the profile input here.

I just didn't want that nonexistent profile config to lead to an unexpected error in deployed env.